src/actions/actionTypes.js

```diff
@@ -24,5 +24,8 @@ export const USER_LOGIN_SUCCESS = 'USER_LOGIN_SUCCESS';
 export const USER_LOGIN_ERROR = 'USER_LOGIN_ERROR';
 export const USER_LOGOUT_SUCCESS = 'USER_LOGOUT_SUCCESS';
 
-export const SUBMIT_ERROR = 'SUBMIT_ERROR';
-export const SUBMIT_SUCCESS = 'SUBMIT_SUCCESS';
+export const AUTHOR_SUBMIT_ERROR = 'AUTHOR_SUBMIT_ERROR';
+export const AUTHOR_SUBMIT_SUCCESS = 'AUTHOR_SUBMIT_SUCCESS';
+export const AUTHOR_UPDATE_DATA_REQUEST = 'AUTHOR_UPDATE_DATA_REQUEST';
```

Comment:
``AUTHOR_UPDATE_DATA`` seems weird maybe ``AUTHOR_DATA_*``?

Reply:
actually I was about to go with `AUTHOR_UPDATE_FORM_DATA` seemed too long... I think it's good indicate it's update form data somehow but agree that this is not the best name.

Reply:
``AUTHOR_UPDATE_FORM_DATA`` 👍